src/main/kotlin/nl/info/zac/app/klant/KlantRestService.kt

```diff
@@ -71,13 +72,16 @@ class KlantRestService @Inject constructor(
     @GET
     @Path("persoon/{bsn}")
     fun readPersoon(
-        @PathParam("bsn") @Length(min = 8, max = 9) bsn: String
+        @PathParam("bsn") @Length(min = 8, max = 9) bsn: String,
+        @HeaderParam("X-Verwerking") xVerwerking: String
```

Comment:
```suggestion
        @HeaderParam("X-Verwerking") process: String
```
And I would use a constant instead of a literal string